<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>colormatch</class>
 <widget class="QWidget" name="colormatch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>49</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>49</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>49</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="horizontalWidget_24" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
border-radius:5px;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_20">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="gridWidget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	border-radius:3px;
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_color_black">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(12, 12, 12);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_color_brightBlack">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(118, 118, 118);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_color_red">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(197, 15, 31);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_color_green">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(19, 161, 14);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_color_brightRed">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(231, 72, 86);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_color_brightGreen">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(22, 198, 12);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_color_brightYellow">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(249, 241, 165);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_color_yellow">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(193, 156, 0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_color_blue">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 55, 218);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_color_brightBlue">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(59, 120, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="label_color_purple">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(136, 23, 152);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QLabel" name="label_color_brightCyan">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(97, 214, 214);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QLabel" name="label_color_cyan">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(58, 150, 221);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_color_brightPurple">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(180, 0, 158);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QLabel" name="label_color_white">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(204, 204, 204);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QLabel" name="label_color_brightWhite">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #ffffff;
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_name">
        <property name="font">
         <font>
          <family>Cascadia Code</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Campbell</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_64">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
